src/Symfony/Component/Runtime/autoload.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Component\Runtime\SymfonyRuntime;
+
+if (true === require_once($_SERVER['APP_VENDOR_DIR'] ?? dirname(__DIR__, 2)).'/autoload.php') {
+    return;
+}
+
+if (!isset($_SERVER['APP_RUNTIME'])) {
+    $runtime = new SymfonyRuntime($_SERVER['APP_RUNTIME_OPTIONS'] ?? []);
+} elseif (is_string($runtime = $_SERVER['APP_RUNTIME'])) {
+    $runtime = new $runtime($_SERVER['APP_RUNTIME_OPTIONS'] ?? []);
+}
```

Comment:
If `$_SERVER['APP_RUNTIME']` is not string, then maybe we should throw an exception. Currently we will fail with `called ->resolve() on undefined variable $runtime`.

Reply:
> Currently we will fail with called ->resolve() on undefined variable $runtime

The variable is always defined since it's set in the "if". I just added a check to ensure it's an instance of `RuntimeInterface` if it's not a string.

Reply:
Yepp. You are correct. Thank you.